pkg/plugins/context.go

```diff
@@ -46,15 +46,12 @@ func (ctx *cbContext) Send(msg *pb.Message) error {
 		return err
 	}
 
-	// add some metadata
-	m.Metadata.Src(ctx.plugin.runtime.Name)
-
-	b, err := json.Marshal(msg)
+	b, err := json.Marshal(m)
 	if err != nil {
 		return err
 	}
 
-	if err := sc.Publish(ctx.plugin.runtime.Inbox, b); err != nil {
+	if err := sc.Publish(ctx.plugin.runtime.Outbox, b); err != nil {
```

Comment:
👍 nice catch here, we should get a test for this reply functionality, forgot about it when writing the plugins tests